src/components/Footer/Footer.js

```diff
@@ -1,7 +1,48 @@
 import React from 'react';
-
+import workoutImage from './assets/workoutImage.jpg';
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div
+      className=" sm:h-auto lg:h-64 flex lg:flex-row xl:flex-row flex-col md:flex-col justify-between items-center font-poppins text-secondary"
+      style={{
+        backgroundImage: `url(${workoutImage})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+      }}
+    >
+      <h1 className="text-4xl p-5 select-none">
+        Train to <span className="border-b-4 border-primary">Maintain</span>
+      </h1>
+      <ul className="text-2xl sm:p-2 p-2">
+        <li>
+          <a href="#">Home</a>
```

Comment:
you should use links here to navigate through the router, checkout Navbar branch for more details